pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/guide.en-us.md

```diff
@@ -0,0 +1,1278 @@
+    ```
+
+    **Note:**
+
+    A `specific` version for the `TrilioVault-Operator` Helm chart is used. In this case `2.6.4` is picked, which maps to the `2.6.4` version of the application (see the output from `Step 2.`). It’s good practice in general, to lock on a specific version. This helps to have predictable results, and allows versioning control via `Git`.
+
+Now, please check your `TVK` deployment:
+
+```shell
+helm ls -n tvk
+```
+
+The output looks similar to the following (`STATUS` column should display `deployed`):
+
+```text
+NAME                    NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                           APP VERSION
+triliovault-operator    tvk             1               2021-12-21 08:13:41.644264863 +0000 UTC deployed        k8s-triliovault-operator-2.6.4  2.6.4
+```
+
+Next, verify that `TrilioVault-Operator` is up and running:
+
+```shell
+kubectl get deployments -n tvk
+```
+
+The output looks similar to the following (deployment pods must be in the `Ready` state):
+
+```text
+NAME                                		READY   UP-TO-DATE   AVAILABLE   AGE
+triliovault-operator-k8s-triliovault-operator   1/1     1            1           3d
+```
+
+Now, you can install `TrilioVault Manager Application` using yaml definition (pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/assets/manifests/triliovault-manager-v2.6.4.yaml)
+
+Run the `kubectl` command to install the `Triliovault Manager` using the yaml
+
+```shell
+kubectl apply -f assets/manifests/triliovault-manager-v2.6.4.yaml -n tvk
+```
+
+Now, please check your `TVK` deployment: 
+
+```shell
+helm ls -n tvk
+```
+
+The output looks similar to the following (`STATUS` column should display `deployed`):
+
+```text
+NAME		NAMESPACE	REVISION	UPDATED                                 STATUS		CHART			APP VERSION
+tvk-tvk         tvk             1               2021-12-21 08:18:06.817766759 +0000 UTC deployed        k8s-triliovault-2.6.4   2.6.4
+```
+
+Next, verify that `TrilioVault` is up and running:
+
+```shell
+kubectl get deployments -n tvk
+```
+
+The output looks similar to the following (all deployments pods must be in the `Ready` state):
+
+```text
+NAME                                            READY   UP-TO-DATE   AVAILABLE   AGE
+k8s-triliovault-admission-webhook               1/1     1            1           3d
+k8s-triliovault-control-plane                   1/1     1            1           3d
+k8s-triliovault-exporter                        1/1     1            1           3d
+k8s-triliovault-ingress-gateway                 1/1     1            1           3d
+k8s-triliovault-web                             1/1     1            1           3d
+k8s-triliovault-web-backend                     1/1     1            1           3d
```

Comment:
missing the line:
`triliovault-operator-k8s-triliovault-operator   1/1     1            1           20m`

Reply:
Thank you for pointing this out @raclepoulpe. There is a major change done related to the product installation from v2.6.5. I have updated the installation section which should address this as well.